sdk-workflows/pom.xml

```diff
@@ -0,0 +1,152 @@
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr</groupId>
+    <artifactId>dapr-sdk-parent</artifactId>
+    <version>1.9.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>dapr-sdk-workflows</artifactId>
+  <packaging>jar</packaging>
+  <version>1.9.0-SNAPSHOT</version>
+  <name>dapr-sdk-workflows</name>
+  <description>SDK for Workflows on Dapr</description>
+
+  <properties>
+    <maven.deploy.skip>false</maven.deploy.skip>
+    <grpc.version>1.42.1</grpc.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.dapr</groupId>
+      <artifactId>dapr-sdk</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+      <dependency>
+          <groupId>com.github.gmazzo</groupId>
+          <artifactId>okhttp-mock</artifactId>
+          <version>1.4.1</version>
+          <scope>test</scope>
+      </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.5.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-cli</groupId>
+      <artifactId>commons-cli</artifactId>
+      <version>1.4</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <version>2.7.8</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-testing</artifactId>
+      <version>${grpc.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.microsoft</groupId>
+      <artifactId>durabletask-client</artifactId>
+      <version>1.1.0</version>
+    </dependency>
```

Comment:
I'm not very familiar with pom.xml, but it seems there's a lot of duplication between this pom.xml and `sdk-workflows/pom.xml` in the parent directory. Is this intentional?

Reply:
I believe it is, but as it is a direct port of the file in `sdk-actors` there could be unnecessary duplication in both. Would love to know generally if there are anti-patterns or dated practices in `sdk-actors` that we should be wary of proliferating.

Reply:
Each package needs to declare its own dependencies, but there were deps in actors that were not needed by workflows. Those have been cleaned up.